inc/settings.php

```diff
@@ -284,6 +284,12 @@ function settings_fields( $fields ) {
 			'description' => __( 'How many pixels of scrolling result in a single pixel of parallax motion. 0 means automatic. Lower values give more noticeable effect.', 'siteorigin-panels' ),
 		);
 
+		$fields['general']['fields']['parallax-mobile'] = array(
+			'type'        => 'checkbox',
+			'label'       => __( 'Disable Background Parallax On Mobile', 'siteorigin-panels' ),
+			'description' => __( 'Disable row/widget background parallax when browser is smaller than the mobile width.', 'siteorigin-panels' ),
```

Comment:
Let's perhaps change this to: `Disable row/widget background parallax when the browser is smaller than the mobile width.`

It's quite long already so adding `the` won't hurt I don't think.